packages/api-request-builder/src/build-query-string.js

```diff
@@ -1,7 +1,7 @@
 /* @flow */
 import type {
   ServiceBuilderDefaultParams,
-} from 'types/sdk'
+} from '../../../types/sdk'
```

Comment:
@wizzy25 it is a sublime specific error. Info how to fix it [here](https://github.com/benmosher/eslint-plugin-import/blob/master/README.md#sublimelinter-eslint)